packages/@uppy/provider-views/src/index.js

```diff
@@ -422,7 +422,7 @@ module.exports = class ProviderView {
   }
 
   handleAuth () {
-    const authState = btoa(JSON.stringify({ origin: location.origin }))
+    const authState = btoa(JSON.stringify({ origin: location.origin, redirect: 'http://localhost:3' }))
     const link = `${this.provider.authUrl()}?state=${authState}`
```

Comment:
`http://localhost:3` 🤔

Reply:
yeah that's not meant to be there. Removed